dts/bindings/lora/st,stm32wl-subghz-radio.yaml

```diff
@@ -12,3 +12,37 @@ properties:
     required: true
     description: |
       Position of the "Radio IRQ, Busy" interrupt line.
+
+  power-amplifier-output:
+    type: string
+    required: true
+    description: |
+      Selects between the low- and high-power power amplifier output pin.
+    enum:
+      - "rfo-lp"
+      - "rfo-hp"
+
+  rfo-lp-max-power:
+    type: int
+    default: 15
```

Comment:
I would agree.

The AN5457 states:
![image](https://user-images.githubusercontent.com/17674105/233828806-d76a010c-f888-4709-88b9-fd667b687114.png)

`0x04` corresponds to 14, so the 15 is not a safe default for all boards.

Reply:
> It is mandatory for default values to have an explanation under `description:`

Thanks for the heads up, will add to the description.

> I suspect these properties would be better off without `default` and adding `required: true`

That'd require defining both output `max-power` properties for boards that will often only have one output connected, leading to a superfluous and erroneous DTS property, which I think would be more confusing. The other alternative of using some weird array combinations instead of two enums, or an additional per-property disabled value would be even clunkier I think.

> `0x04` corresponds to 14, so the 15 is not a safe default for all boards.

15 is the default in the STM32CubeWL, but there's probably some additional safeguard logic preventing that setting on 433 MHz boards. At the very least the `*_MAXPOWER` defines are mandatory there to prevent accidentally setting +15 dBm for those.

I'll change the defaults to 14 to be on the safe side, good call :)

Reply:
Makes sense. Thanks for the final update!